raft/raftpb/raft.proto

```diff
@@ -67,7 +67,8 @@ message Message {
 	optional Snapshot    snapshot    = 9  [(gogoproto.nullable) = false];
 	optional bool        reject      = 10 [(gogoproto.nullable) = false];
 	optional uint64      rejectHint  = 11 [(gogoproto.nullable) = false];
-	optional bytes       context     = 12;
+  	optional bytes       context     = 12;
+  	optional uint64      commitTerm  = 13 [(gogoproto.nullable) = false];
```

Comment:
It's surprising that `commitTerm` is set only on vote messages, and not on all messages for which `commit` is set. Should we be setting it everywhere for consistency? Or maybe it's better to omit it when we can since we have to handle that case for backwards compatibility. 

If we do limit `commitTerm` to vote messages alone, we should add a comment about it. (I know we're not good about comments in this file so far, but we should start moving in the right direction)